go/vt/discovery/healthcheck.go

```diff
@@ -165,6 +165,7 @@ func init() {
 	}
 
 	servenv.OnParseFor("vtctld", registerWebUIFlags)
+	servenv.OnParseFor("vttablet", registerWebUIFlags)
```

Comment:
`registerWebUIFlags` is a terrible func name, all but 1 flag are not web UI related. But wasn't me 🤦